pkg/blob/blob.go

```diff
@@ -547,7 +548,7 @@ func (d *Driver) GetAuthEnv(ctx context.Context, volumeID, protocol string, attr
 		authEnv = append(authEnv, "AZURE_STORAGE_SPN_CLIENT_SECRET="+storageSPNClientSecret)
 	}
 
-	return rgName, accountName, accountKey, containerName, authEnv, err
+	return rgName, accountName, secretName, containerName, authEnv, err
```

Comment:
I suspect this change will break non edgecache protocol volumes. Maybe you should just add secretName to the returned values?

Reply:
fixed in the new update, returning both secret and secretNamespace now as new values